charts/aws-cloud-controller-manager/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
```

Comment:
Can we include appVersion as well? `"v1.19.0-alpha.1"`

Reply:
Sure @ayberk I will add appVersion